docs/guide/personalization/developer_guide/tracking_api.md

```diff
@@ -171,7 +171,7 @@ For a list of embedded parameters that each of the events may use, see the follo
 
 |Name|Description|Values|
 |---|---|---|
-|`customerid`|A customer ID (for example "00000"). Can be used to identify a website in installations that [hosts multiple SiteAccesses]([[= user_doc =]]/personalization/use_cases/#multiple-website-hosting).|alphanumeric|
+|`customerid`|A customer ID (for example "00000"). Can be used to identify a website in installations that [hosts multiple SiteAccesses]([[= user_doc_4_6 =]]/personalization/use_cases/#multiple-website-hosting).|alphanumeric|
```

Comment:
https://github.com/ibexa/documentation-user/blob/4.6/docs/personalization/use_cases.md
file could replace
https://github.com/ibexa/documentation-user/blob/3.3/docs/personalization/use_cases.md

Reply:
`user_doc_4_6` links could be fixed on both user and dev doc sides in another PR.